composer.json

```diff
@@ -32,6 +32,7 @@
         "soundasleep/html2text": "^1.1",
         "spatie/calendar-links": "^1.2",
         "spinen/laravel-discourse-sso": "^1.3",
+        "symfony/dom-crawler": "^5.2",
```

Comment:
Should this not be in `require-dev` ?  (i.e. only needed for tests?)

Reply:
Yes - pushed.